daemon/src/model/cfd.rs

```diff
@@ -982,11 +1089,22 @@ impl Cfd {
                 // TODO: Deal with failed contract setup
                 self.during_contract_setup = false;
             }
+            RolloverStarted => {
+                self.during_rollover = true;
+            }
+            RolloverAccepted => {} /* TODO: It's weird we have no state change for applying an */
```

Comment:
@thomaseizinger @da-kami: I'm removing this TODO because I hate all code TODOs, but the point stands. Is this a bad sign? The `RolloverAccepted` was only introduced because a different aggregate would care (in `projection`). Maybe that's fine.